generateData/coreDBQueries.js

```diff
@@ -0,0 +1,62 @@
+const hrtime = require('process.hrtime');
+const pgdb = require('./pgdbConnection.js');
+const csdb = require('./csdbConnection.js');
+
+pgdb.connect();
+// pick a database to test (between 'postgres' and 'cassandra')
+const selectedDataBase = 'cassandra';
+const noOfQueries = 1;
+const reviewId = 33;
+
+async function csExecuteOneAtATime(info, timer) {
+  const query = 'SELECT * FROM reviews WHERE id=?;';
+  const options = { prepare: true };
+
+  while (info.counter++ < info.totalLength) {
+    const params = [reviewId];
+    const result = await csdb.execute(query, params, options);
+    if (result.rows[0]) {
+      await console.log(`${selectedDataBase} query took ${hrtime(timer, 's')}seconds`);
```

Comment:
Hmmm...for this await...I don't see anything async happening -- is it hrtime?

Reply:
I think I was getting some error. It was working for small queries and it was warning me not to put awaits in a loop.